python_otbr_api/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""API to interact with the Open Thread Border Router REST API."""
+from __future__ import annotations
+from http import HTTPStatus
+
+import aiohttp
+
+
+class OTBRError(Exception):
+    """Raised on error."""
+
+
+class OTBR:
+    """Class to interact with the Open Thread Border Router REST API."""
+
+    def __init__(
+        self, url: str, session: aiohttp.ClientSession, timeout: int = 10
+    ) -> None:
+        """Initialize."""
+        self._session = session
+        self._url = url
+        self._timeout = timeout
+
+    def _raise_for_status(self, response: aiohttp.ClientResponse) -> None:
```

Comment:
Not used. Btw you can pass raise_for_status also to the get and post functions

Reply:
Removed